src/scenarios/find-slot.ts

```diff
@@ -34,10 +35,27 @@ export const findSlot = async (wd: WebDriver): Promise<boolean> => {
       wd,
       By.xpath('//*[contains(text(),\'Appointment selection\')]')
     );
-    console.log(`[findSlot]: appointment found`);
-    return true;
+    console.log(`[findSlot]: found calendar`);
+    return !await existsErrorBox(wd)
   } catch (e) {
     console.error(`[findSlot]: appointment not found ${e}`);
     return false;
   }
 };
+
+async function existsErrorBox(wd: WebDriver) {
+  try {
+    // const dateSelector = '//*[@id="xi-fs-2"]';
+    const box = await Utils.waitUntilVisible(
+      wd,
+      By.xpath('//*[@id="messagesBox"]/ul/li'),
+      SMALL_TIMEOUT
```

Comment:
maybe it's worth to rename it to something like ERR_MESSAGE_TIMEOUT to more explicit